notebooks/11-test-verifications-to-get-human-labels.ipynb

```diff
@@ -0,0 +1,6704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.     }
</pre><p>would just 30 samples be sufficient to 95% of confidence? TBH I don't remember very well of statistics </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bleu/farcaster-social-graph/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>What change here are the limits of our 95% confidence, but the analysis of these 30 samples are just providing to us a small idea of the true proportion. The conclusion would be something like: "after checking these 30 samples, we are 95% confident that the proportion for all samples is between (x-15%) and (x + 15%)", where x is the proportion observed.</p><p> </p><p>Despite this is not the ideal, I think it would be not feasible to look at, for example, 100 samples just to decrease this uncertainty limits to 5%</p>